### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sam-github**

### CODE REVIEW

**Comment by kjdelisle** [comment]
LGTM

**Comment by pthieu** [comment]
Async 1.x to 2.x, not sure if we are using any uncommon methods that have changed their signature, suggest checking.

File: package.json [hunk 1]
```diff
@@ -37,10 +37,11 @@
     "test": "tap --timeout 300 test/test-*.js"
   },
   "dependencies": {
-    "async": "^1.0.0",
+    "async": "^2.0.0",
+    "byline": "^5.0.0",
```
Comment (rmg) [inline package.json]: Looks like you did `--save` instead of `--save-dev` when you upgraded byline. It is still in `devDependencies` as `^4.2.1`.

**Comment by rmg** [comment]
@slnode test please

**Comment by sam-github** [comment]
the only async we use is async.map, I've reviewed, all usages are good (and it hasn't changed)

File: package.json [hunk 2]
```diff
@@ -38,10 +38,11 @@
   },
   "dependencies": {
     "appmetrics": "^1.0.13",
-    "async": "^1.0.0",
+    "async": "^2.0.0",
+    "byline": "^5.0.0",
```
Comment (rmg) [inline package.json]: Wasn't this supposed to replace `byline` in the `devDependencies`?

**Comment by rmg** [comment]
@sam-github I think github ate my previous comment and it got missed before merging.. `byline` is now listed in both `dependencies` and `devDependencies`, but with different version specs.

**Pull request merged**